mockld/crypto/config.go

```diff
@@ -0,0 +1,6 @@
+package crypto
```

Comment:
Everything in crypto was copied from streamer (changing only the imports and removing some feature flagging stuff we don't need).